docs/Users/Onboarding.md

```diff
@@ -4,11 +4,11 @@ pagination_next: Contact-Us
 
 # Onboarding
 
-When you visit an app that uses Vincent for Authentication you'll be directed to a Consent Page where you can review the permissions requested by the application and provide precisely scoped delegation to it. Follow the following steps to onboard to the Vincent ecosystem:
+Each time you visit an app on Vincent, you’ll be directed to that app’s dedicated consent page. This is where you will configure and manage your app-specific delegations, permitting it to use your assets and data according to the guidelines you set. Follow the steps below to begin your journey.
 
 ## 1. Create an Agent Wallet
 
-The first time you visit an app that uses Vincent for Authentication & Delegation you'll be prompted to create an Agent Wallet with an Authmethod of your choice. Note, this wallet is a non-custodial wallet managed by the decentralized Lit Protocol network. It is **not** managed by the app you're visiting thus only you have access to it.
+The first time you visit an app powered by Vincent, you’ll be prompted to create your Agent Wallet using a credential of your choice. Currently, email, phone number, and Passkey methods are supported. This credential is used to control your Agent Wallet which is securely managed by Lit Protocol’s [decentralized key management network](https://developer.litprotocol.com/resources/how-it-works). You will have the ability to use the same Agent Wallet across all Vincent apps, or create a new one if you choose to.
```

Comment:
> or create a new one if you choose to.

I don't think we support generating a new wallet using the same Auth Method, FYI. The preference was for a user to only ever have a single Agent Wallet used for every Vincent App